modules/core/organization/src/Plugin/Action/OrganizationClone.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Drupal\organization\Plugin\Action;
+
+use Drupal\Core\Action\Plugin\Action\EntityActionBase;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\organization\Entity\OrganizationInterface;
+
+/**
+ * Action that clones an organization.
+ *
+ * @Action(
+ *   id = "organization_clone_action",
+ *   label = @Translation("Clone an organization"),
+ *   type = "organization"
+ * )
+ */
+class OrganizationClone extends EntityActionBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function execute(OrganizationInterface $organization = NULL) {
+    if ($organization) {
+      $cloned_organization = $organization->createDuplicate();
+      $new_name = $organization->getName() . ' ' . $this->t('(clone of organization #@id)', ['@id' => $organization->id()]);
+      $cloned_organization->setName($new_name);
+      $cloned_organization->save();
```

Comment:
We should also set the owner UID, see #851 

But more generally, kinda wonder how useful a `clone` action is for organizations :thinking: doesn't hurt

Reply:
Added this here: https://github.com/paul121/farmOS/commit/4b3794a9b79e0038e1beb9defba317160f6c24de